### PR TITLE
Menu: Added extra examples + explanation

### PR DESCRIPTION
**Pull request by Yinzy00**
## Description

I added extra examples + explanation to the menu docs for V14 and v15

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
V14, V15

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for this @Yinzy00 🙌 🎉 

I've forwarded this to our developers, and I'll make sure we get it reviewed as soon as possible.

**Review by Frost117 — APPROVED** [review]
@sofietoft 

I ran @Yinzy00 code examples locally and they work on my machine. LGTM :)

File: 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md [hunk 1]
```diff
@@ -4,37 +4,173 @@
 This page is a work in progress and may undergo further revisions, updates, or amendments. The information contained herein is subject to change without notice.
 {% endhint %}
 
-## Menu
-
 <figure><img src="../../../.gitbook/assets/menu.png" alt="" width="250"><figcaption><p>Menu</p></figcaption></figure>
 
-```typescript
+**JSON Manifest:**
```
Comment (sofietoft) [inline 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md:9]: The two code snippets here - what exactly to they show? 😅 
Would be good to add a little text explaining 🙌

File: 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md [hunk 2]
```diff
@@ -4,37 +4,173 @@
...
+        meta: {
+            label: 'My Menu Item',
+            menus: ["My.Menu"]
+        },
+        element: () => import('./menu-items.ts')
+    }
+];
+```
+
+### The UI Element
```
Comment (sofietoft) [inline 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md:73]: I've changed this from "The Lit Element" to "The UI Element".

Makes it a little more generic, and easier to understand for those not familiar with Lit.
What do you think?

**Review by sofietoft — CHANGES_REQUESTED** [review]
I've taken the liberty to correct some grammar 💪 

Also, I've added a couple of comments. 
Please let me know if something doesn't make sense!

Regarding the code snippets, I think it would be great if you could wrap them using the GitBook formats, so we can highlight the file name a little easier.
See the docs for reference: https://docs.gitbook.com/content-editor/blocks/code-block#representation-in-markdown

File: 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md (hunk 2 above)
Comment (Yinzy00) [reply 14/umbraco-cms/customizing/extending-overview/extension-types/menu.md:73]: I will look in to it tonight!

**Review by sofietoft — APPROVED** [review]

**Comment by sofietoft** [comment]
Looks like everything checks out here 🎉 

Thanks again for putting in the effort to help us improve the Umbraco documentation.
MUCH appreciated 👏 

I'll get this one merged!

**Pull request merged**